routes/spentRoutes.js

```diff
@@ -0,0 +1,38 @@
+var db = require("../models");
+
+module.exports = function(app) {
+  app.get("/api/users/spent/:userid/", function(req, res) {
+    db.Spent.findAll({
+      where: {
+        id: req.params.userid
```

Comment:
I checked this in Postman and it will need to be UserId: req.params.userid to pull back all rows associated with that user. The way it's written will pull back the first row only since it's just finding the one where id in the spent table is the param number.

Reply:
Good catch. I fixed it

Reply:
Cool...for some reason the change isn't showing for me.